### ISSUE TITLE
Rounding behaves differently with or without the number of decimals.

### ISSUE THREAD

**Issue by jpkolbush** [josdejong/mathjs#1404]
Hello,

I found that rounding behaves differently depending on if you pass in 0 or not.

If you evaluate `mathjs.round(-2.5)` it returns -2
However if you evaluate `mathjs.round(-2.5, 0)` it returns -3

We wish to use the later case (always rounding away from 0) so its not a pressing issue, as we can just pass in the extra parameter. But I figured it should be brought to your attention.

**Comment by josdejong** [comment]
That's odd, I would indeed want consistent behavior there. I would expect `mathjs.round(-2.5)` to return `-3` indeed. Thanks for reporting.

Anyone interested in digging into this issue?

**Comment by Yaffle** [comment]
`Math.round(-2.5) === -2`

**Comment by josdejong** [comment]
Turns out `round(x)` did simply use the built-in `Math.round` instead of the method used for `round(x, n)`, which allowed for inconsistent behavior. 

See fix: https://github.com/josdejong/mathjs/commit/7d00987a3fbe7f7b75e61151a8b228d8a56fb8d6

Both numbers and BigNumbers now have the following behavior (similar to Matlab and Python, and unlike Mathematica which rounds -2.5 to 2):

```js
math.round(2.5) // number 3
math.round(-2.5) // number -3
math.round(math.bignumber(2.5)) // BigNumber 3
math.round(math.bignumber(-2.5)) // BigNumber -3
```

**Comment by josdejong** [comment]
Should be fixed now in `v5.5.0`.
